taichi/program/launch_context_builder.h

```diff
@@ -102,6 +102,7 @@ class LaunchContextBuilder {
   const StructType *args_type{nullptr};
   size_t result_buffer_size{0};
   bool has_grad[taichi_max_num_args_total];
+  uint64 cc_args[taichi_max_num_args]{0};
```

Comment:
agh sad...
shall we add a TODO here to make sure we remove it along with cc backend?